lib/webdriveragent.js

```diff
@@ -108,6 +110,18 @@ class WebDriverAgent {
     });
   }
 
+  async checkForDependencies() {
+    if (await fs.hasAccess(this.bootstrapPath + "/Carthage")) {
+      this.xcodebuild.start();
+      return;
+    }
+    let proc = new SubProcess('/bin/bash', ['Scripts/bootstrap.sh'], {cwd: this.bootstrapPath});
```

Comment:
I still think we shouldn't be using `SubProcess` to call `bootstrap.sh`.

And probably not awaiting `xcodebuild.start()` is a bug.